tox.ini

```diff
@@ -21,15 +21,14 @@ commands = coverage combine
 [testenv:style]
 skip_install = True
 deps = ruff
-commands = ruff check --diff
-           ruff check --output-format=full
+commands = ruff check
```

Comment:
`ruff` does the "right thing" (TM) now by default on `ruff check` (default is `full`)